service/test/_registry_test.js

```diff
@@ -796,9 +817,22 @@ describe('Registry', function() {
       <stringified array supplied by the argument devices annotated with importMode property and deviceId property replaced by id>
       ```
       ]*/
+      /* Tests_SRS_NODE_IOTHUB_REGISTRY_06_029: [** A device information with an authentication object that contains a `type` property is considered normalized.] */
+      /* Tests_SRS_NODE_IOTHUB_REGISTRY_06_031: [A device information with an authentication object that doesn't contain the x509Thumbprint property will be normalized with a `type` property with value "sas".] */
+      /* Tests_SRS_NODE_IOTHUB_REGISTRY_06_030: [A device information with an authentication object that contains the x509Thumbprint property will be normalized with a `type` property with value "selfSigned".] */
       it('constructs a valid HTTP request', function(testCallback) {
-        var addedDevices = [{deviceId: 'deviceone'},{deviceId: 'devicetwo'}];
-        var sentBody = [{id:'deviceone',importMode:'create'},{id:'devicetwo',importMode:'create'}];
+        var addedDevices = [
```

Comment:
I respectfully disagree.

First, adding the sentBody into the device description really complicates from a reading perspective what we have here.

Second, each of these paired things is restricted to a single test.  It is easily read that via their contents that they are linked.

Third, this not something new just thought up.  This paradigm was already in the tests. 

---
In reply to: [128303235](https://github.com/Azure/azure-iot-sdk-node/pull/61#discussion_r128303235) [](ancestors = 128303235)